target/linux/ath79/dts/ar9344_mikrotik_routerboard-2011.dts

```diff
@@ -0,0 +1,217 @@
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partition@0 {
+					label = "bootloader1";
+					reg = <0x0 0x0>;
+					read-only;
+				};
+
+				hard_config: hard_config {
+					read-only;
+				};
+
+				bios {
+					size = <0x1000>;
+					read-only;
+				};
+
+				soft_config {
+				};
+			};
+		};
+	};
+};
+
+&nand {
+	status = "okay";
+
+	nand-ecc-mode = "soft";
+	qca,nand-swap-dma;
+	qca,nand-scan-fixup;
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "booter";
+			reg = <0x0000000 0x0040000>;
+			read-only;
+		};
+
+		partition@40000 {
+			label = "kernel";
+			reg = <0x0040000 0x03c0000>;
+		};
+
+		partition@400000 {
+			label = "ubi";
+			reg = <0x0400000 0x7c00000>;
+		};
+	};
+};
+
+&uart {
+	status = "okay";
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy-mask = <0>;
+
+	phy0: ethernet-phy@0 {
+		reg = <0>;
+		phy-mode = "rgmii-id";
+
+		qca,ar8327-initvals = <
+			0x04 0x07800000 /* PORT0 PAD MODE CTRL */
+			0x10 0x81000080 /* POWER_ON_STRAP */
+			0x50 0xc731c731 /* LED_CTRL0 */
+			0x54 0x00000000 /* LED_CTRL1 */
+			0x58 0x00000000 /* LED_CTRL2 */
+			0x5c 0x0030c300 /* LED_CTRL3 */
+			0x7c 0x0000007e /* PORT0 STATUS */
+			0x0c 0x00000080 /* PORT6 PAD Mode */
+			0xe0 0xc70167d0 /* SGMII_CTRL */
+			/*0x94 0x0000007e PORT6_STATUS */
+		>;
+	};
+};
+
+&eth0 {
+	status = "okay";
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	/* default for ar934x, except for 1000M */
+	pll-data = <0x6f000000 0x00000101 0x00001616>;
+
+	phy-mode = "rgmii-id";
+	phy-handle = <&phy0>;
+
+	/*gmac-config {
+		device = <&gmac>;
+		rgmii-enabled = <1>;
+		rgmii-gmac0 = <1>;
+	};*/
+
+	port@0 {
+		compatible = "swconfig,port";
+		reg = <0>;
+		swconfig,segment = "lan";
+		swconfig,portmap = <1 1>;
+	};
+
+	port@1 {
+		compatible = "swconfig,port";
+		reg = <1>;
+		swconfig,segment = "lan";
+		swconfig,portmap = <2 2>;
+	};
+
+	port@2 {
+		compatible = "swconfig,port";
+		reg = <2>;
+		swconfig,segment = "lan";
+		swconfig,portmap = <3 3>;
+	};
+
+	port@3 {
+		compatible = "swconfig,port";
+		reg = <3>;
+		swconfig,segment = "lan";
+		swconfig,portmap = <4 4>;
+	};
+
+	port@4 {
+		compatible = "swconfig,port";
+		reg = <4>;
+		swconfig,segment = "lan";
+		swconfig,portmap = <5 5>;
+	};
+
+	port@5 {
+		compatible = "swconfig,port";
+		reg = <5>;
+		swconfig,segment = "sfp";
+		swconfig,portmap = <6 6>;
+	};
+};
+
+&eth1 {
+	status = "okay";
+
+	phy-handle = <&swphy0>;
+	phy-mode = "gmii";
+
+	gmac-config {
+		device = <&gmac>;
+		switch-phy-swap = <0>;
+		switch-only-mode = <1>;
+	};
+};
+
+&wmac {
+	status = "okay";
+
+	qca,no-eeprom;
+};
+
+&gpio {
+	pinctrl-names = "default";
+
+	gpio_nand_nce {
+		gpio-hog;
+		gpios = <14 0>;
+		output-high;
+		line-name = "rb-2011:nand:nce";
+	};
+};
+
+&usb {
+	status = "okay";
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	hub_port1: port@1 {
+		reg = <1>;
+		#trigger-source-cells = <0>;
+	};
+};
```

Comment:
The port definition is only needed when you want to link a LED to it. Otherwise, just
```
&usb {
	status = "okay";
};
```
should be enough.